### PR TITLE
[15.0][MIG]purchase_cancel_confirm: Migration to 15.0

### PR DESCRIPTION
**Pull request by bizzappdev**

### CODE REVIEW

**Comment by Saran440** [comment]
@bizzappdev Hello, Can you pre-commit this PR and update your code again, please?

This module depend on - https://github.com/OCA/server-ux/pull/541

**Review by PatthamaCH — APPROVED** [review]
OK

**Comment by bizzappdev** [comment]
ready for Review

File: purchase_cancel_confirm/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Purchase Cancel Confirm",
+    "version": "15.0.1.0.1",
```
Comment (Saran440) [inline purchase_cancel_confirm/__manifest__.py]:
```suggestion
    "version": "15.0.1.0.0",
```

File: test-requirements.txt [hunk 2]
```diff
@@ -1 +1 @@
-odoo_test_helper
+odoo-addon-base_cancel_confirm@ git+https://github.com/oca/server-ux@refs/pull/541/head#subdirectory=setup/base_cancel_confirm
```
Comment (Saran440) [inline test-requirements.txt]: please back to standard

**Review by Saran440 — COMMENTED** [review]
Code Reviewed.

**Review by Saran440 — APPROVED** [review]
LGTM :+1:

**Review by HviorForgeFlow — APPROVED** [review]
Functional review

**Pull request merged**